### ISSUE TITLE
deleted "false double entries" in commit

### ISSUE THREAD

**Issue by pexner** [picons/picons#23]
In commit
https://github.com/picons/picons/commit/6d47017523d625f0c89b1af70f8d07169c2df83a
you deleted "false double entries". These aren't doubles.

All these regional channels have the same picon "france3".

Maybe I should have made a more precise description in my issue.
Is this correct then?
srp.index:
```
1001_5014_55F_DDE0000=france3amiens
1002_5140_55F_DDE0000=france3besancon
1003_5078_55F_DDE0000=france3bordeaux
1004_5140_55F_DDE0000=france3caen
1005_5078_55F_DDE0000=france3clermontferrand
1006_5140_55F_DDE0000=france3dijon
1007_5078_55F_DDE0000=france3grenoble
1008_5014_55F_DDE0000=france3lille
1009_5078_55F_DDE0000=france3limoges
1010_5078_55F_DDE0000=france3lyon
1011_5078_55F_DDE0000=france3marseille
1012_5078_55F_DDE0000=france3montpellier
1013_5014_55F_DDE0000=france3nancy
1014_5078_55F_DDE0000=france3nantes
1015_5014_55F_DDE0000=france3nice
1016_5140_55F_DDE0000=france3orleans
1017_5140_55F_DDE0000=france3paris
1018_5014_55F_DDE0000=france3poitiers
1019_5014_55F_DDE0000=france3reims
1020_5078_55F_DDE0000=france3rennes
1021_5140_55F_DDE0000=france3rouen
1022_5014_55F_DDE0000=france3strassbourg
1023_5014_55F_DDE0000=france3toulouse
1024_5014_55F_DDE0000=france3viastella
```
snp.index:
```
france3amiens=france3
france3besancon=france3
france3bordeaux=france3
france3caen=france3
france3clermontferrand=france3
france3dijon=france3
france3grenoble=france3
france3lille=france3
france3limoges=france3
france3lyon=france3
france3marseille=france3
france3montpellier=france3
france3nancy=france3
france3nantes=france3
france3nice=france3
france3orleans=france3
france3paris=france3
france3poitiers=france3
france3reims=france3
france3rennes=france3
france3rouen=france3
france3strassbourg=france3
france3toulouse=france3
france3viastella=france3
```

**Comment by AbuBaniaz** [comment]
I don't understand what you are trying to convey. Can you rephrase?

**Comment by AbuBaniaz** [comment]
We already have these
```
1001_5014_55F_DDE0000=france3picardie
1002_5140_55F_DDE0000=france3franchecomte
1003_5078_55F_DDE0000=france3aquitaine
1004_5140_55F_DDE0000=france3normandie
1005_5078_55F_DDE0000=france3auvergne
1006_5140_55F_DDE0000=france3bourgogne
1007_5078_55F_DDE0000=france3alpes
1008_5014_55F_DDE0000=france3nordpasdecalais
1009_5078_55F_DDE0000=france3limousin
1010_5078_55F_DDE0000=france3rhonealpes
1011_5078_55F_DDE0000=france3cotedazur
1012_5078_55F_DDE0000=france3languedocroussillon
1013_5014_55F_DDE0000=france3lorraine
1014_5078_55F_DDE0000=france3paysdelaloire
1015_5014_55F_DDE0000=france3provencealpes
1016_5140_55F_DDE0000=france3centre
1017_5140_55F_DDE0000=france3parisiledefrance
1018_5014_55F_DDE0000=france3poitoucharentes
1019_5014_55F_DDE0000=france3champagneardenne
1020_5078_55F_DDE0000=france3bretagne
1021_5140_55F_DDE0000=france3normandie
1022_5014_55F_DDE0000=france3alsace
1023_5014_55F_DDE0000=france3midipyrenees
1024_5014_55F_DDE0000=france3corse
```
These are what the picons ARE already.
Are you saying they are wrong and SHOULD be what you posted instead?

**Comment by pexner** [comment]
Let me check. Give me some time...

**Comment by pexner** [comment]
OK.
Mea culpa.
I'll close.

**Comment by Lexzie** [comment]
That's why I said next time check if references are already present.
Simpliest check is if your channels have a picon or not.
If so, you can check if they are okay. If not you can submit the correct ones.
In this case they were okay, only `france3nouvelleaquitaine` was missing.
I didn't make all those logos and links for nothing.

**Comment by AbuBaniaz** [comment]
The links @pexner posted are different to what we have.

**Comment by Lexzie** [comment]
That does not make them the right ones.
I checked them (and even added a new one).
